saleor/order/models.py

```diff
@@ -418,7 +418,9 @@ class OrderLine(models.Model):
     variant_name = models.CharField(max_length=255, default="", blank=True)
     translated_product_name = models.CharField(max_length=386, default="", blank=True)
     translated_variant_name = models.CharField(max_length=255, default="", blank=True)
-    product_sku = models.CharField(max_length=255)
+    product_sku = models.CharField(max_length=255, null=True, blank=True)
+    # str with GraphQL ID used as fallback when product SKU is not available
+    product_id = models.CharField(max_length=255, null=True, blank=True)
```

Comment:
Wouldn't be useful to have here a foreign key relation instead of graphql's product ID? You can always convert DB id into a graphql ID but relation with the product would be useful.

Reply:
also (I can think wrongly here) django uses field like `variant_id` or `product_id` when you define new relation like `variant` or `product`. Would adding this field as `product_id`  won't break this approach?

Reply:
Changing this field to foreign key will defeat the point as it's supposed to preserve GraphQL ID of purchased variant even after this variant has been deleted from the database. But I'll rename this field to something else (perhaps `product_code`) so it won't potentially conflict with `product_id`.

Reply:
I've tried few names but I've settled on `productVariantId` because there are already few places in Saleor that use this approach.